core/src/modules/ops/setting.ts

```diff
@@ -160,14 +160,6 @@ export namespace SettingOps {
         "How many days should we keep Import records for on the server?",
       type: "number",
     },
-    {
-      key: "sweeper-delete-old-exports-days",
-      title: "Sweeper: Delete Old Exports Days",
-      defaultValue: 31,
-      description:
-        "How many days should we keep Export records for on the server?  We will keep the most recent export for each Record & Destination.",
-      type: "number",
-    },
```

Comment:
This setting is removed now - we keep the most recent record-destination export for 90 days.

Reply:
Does it now clear out the other ones at some point?